app/views/layouts/_nav.html.erb

```diff
@@ -1,6 +1,8 @@
-<nav class="splash w-full flex gap-20 text-white p-6 text-xl">
-  <p><%= link_to '<i class="fa-solid fa-bars"></i>'.html_safe, root_path %></p>
+<nav class="splash w-full flex gap-12 text-white p-6 text-xl">
+  <p><%= link_to '<i class="fa-solid fa-bars"></i>'.html_safe, unauthenticated_root_path %></p>
```

Comment:
- [ ] The changes you made here are not being seen on the live version of your project. It still uses the hamburger button as the back button as seen in the image below. You could also evenly space the buttons and the heading Transaction
![image](https://user-images.githubusercontent.com/45460192/220406736-e0c87997-d0b6-4e83-b466-2a8733e3152b.png)